src/DotLiquid.Tests/Filters/StandardFiltersLatestFrenchTests.cs

```diff
@@ -0,0 +1,21 @@
+using System;
+using System.Globalization;
+using NUnit.Framework;
+
+namespace DotLiquid.Tests.Filters
+{
+    [TestFixture]
+    public class StandardFiltersLatestFrenchTests : StandardFiltersV24Tests
```

Comment:
Is this a placeholder to add future tests that shows any output difference with French locale?

Reply:
I think there's an existing bug/unknown around thousands separators, given we use custom Regex for determining if it's integer or real, and then default to InvariantCulture for reals. I will try to investigate what the existing behaviour is. There is inherent ambiguity in there already.

Reply:
Rasied as #600.